README.md

```diff
@@ -5,13 +5,13 @@ The project composes of the mobile version of my personal portfolio. The project
 
 # Live
 
-https://b77748.github.io/arnaud_portfolio/
+https://b77748.github.io/arnaud_portfolio.github.io/
```

Comment:
- [ ] This live link is broken, Kindly change it to `https://b77748.github.io/arnaud_portfolio/`.
`